src/primitives/transaction.cpp

```diff
@@ -12,6 +12,7 @@
 #include <tinyformat.h>
 #include <uint256.h>
 #include <util/strencodings.h>
+#include <util/transaction_identifier.h>
```

Comment:
nit: no longer needed here. In `txorphanage.cpp` and `test/orphanage_tests.cpp`, `#include <util/transaction_identifier.h>` should be replaced with `#include <primitives/transaction.h>` since we need that include already anyway.